HeRoBackEnd/Controllers/RecruitmentController.cs

```diff
@@ -43,6 +43,18 @@ public IActionResult Get(int recruitmentId)
             return Ok(recruitment);
         }
 
+        /// <summary>
+        /// Get a number of Recruitment
+        /// </summary>
+        [HttpGet]
+        [Route("Recruitment/GetQuantity")]
+        public int GetQuantity()
+        {
+            int numberOfRecruitment = service.GetQuantity();
```

Comment:
Everything is wrong in here. You need to return `totalCount` value i Paging object

Reply:
what should I put in Paging object?